numpy/core/include/numpy/ufuncobject.h

```diff
@@ -251,15 +251,21 @@ typedef struct _tagPyUFuncObject {
 #endif
 
 /*
+ * UFunc has unit of 0, and the order of operations can be reordered
+ * This case allows reduction with multiple axes at once.
+ */
+#define PyUFunc_Zero 0
+/*
  * UFunc has unit of 1, and the order of operations can be reordered
  * This case allows reduction with multiple axes at once.
  */
 #define PyUFunc_One 1
 /*
- * UFunc has unit of 0, and the order of operations can be reordered
- * This case allows reduction with multiple axes at once.
+ * UFunc has unit of -1, and the order of operations can be reordered
+ * This case allows reduction with multiple axes at once. Intended for
```

Comment:
That last sentence is not strictly necessary. There could potentially be other ufuncs that use this.